### ISSUE TITLE
Most widgets can't receive keyboard focus

### ISSUE THREAD

**Issue by mwcampbell** [linebender/druid#2095]
On Windows and Linux, all interactive widgets should be able to receive keyboard focus and let the user interact with them using the keyboard. This is particularly important for screen reader accessibility. It's true that screen readers have their own commands for navigating and interacting with widgets even when those widgets don't support keyboard focus. But users generally expect to be able to use Tab and Shift+Tab to move through interactive widgets, Space to activate a button or toggle a checkbox, arrow keys to move a slider, and so on. I understand that this is not the norm on Mac.

**Comment by ratmice** [comment]
I know https://github.com/linebender/druid-widget-nursery has at least "An advanced version of the slider which also allows keyboard input."
